### DISCUSSION TITLE
Feedback: Getting started

### DISCUSSION THREAD

**Discussion by maksnester** [mswjs/mswjs.io#331] [Q&A]
- [Open page](https://mswjs.io/docs/getting-started/)

## Feedback

I'd like to see a link on a small but complete repo with examples that cover both MSW in the browser and node (for tests). 

Offtopic: also a light theme for the docs would be so nice to have x.x

**Comment by kettanaito** [comment]
Hey, @maksnester. Thanks for those suggestions! I think they make perfect sense and would improve the getting started page. 

We do have [Examples](https://github.com/mswjs/examples), just need to add a link to it on the Getting started page. Regarding the light theme, I believe there's already a lot of work done in #269. 

_Edit:_ I've added the [Examples section](https://mswjs.io/docs/getting-started#examples) to the Getting started page. I hope it helps.
